test/oldtest.py

```diff
@@ -58,17 +58,17 @@
 config.read(common.CONFIG_FILE)
   
 cloudverifier_port = config.get('general', 'cloudverifier_port')
-cloudnode_port = config.get('general', 'cloudnode_port')
```

Comment:
this file is all kinds of busted, so i'm not sure how useful it is to keep around.